src/test/java/com/wikia/webdriver/elements/mercury/old/curatedcontent/CuratedMainPagePageObject.java

```diff
@@ -106,4 +98,8 @@ private boolean isCuratedElementVisible(WebElement element) {
       this.value = value;
     }
   }
+
+  private void editArticle() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "editArticle" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1186)